test_runner/fixtures/utils.py

```diff
@@ -613,3 +624,25 @@ def human_bytes(amt: float) -> str:
         amt = amt / 1024
 
     raise RuntimeError("unreachable")
+
+
+def all_pairs_component_versions() -> Iterable[object]:
+    """
+    This function generates all the pairs of old (False) or new (True)
```

Comment:
If we're moving away from bitmasks, I think it makes sense to use use "old" and "new" — this will improve readability a lot

Reply:
done

Reply:
The comment still says False / True; should it also be updated?